sdks/python/setup.py

```diff
@@ -366,7 +366,7 @@ def get_portability_package_data():
           'jsonpickle>=3.0.0,<4.0.0',
           # numpy can have breaking changes in minor versions.
           # Use a strict upper bound.
-          'numpy>=1.14.3,<1.27.0',  # Update pyproject.toml as well.
+          'numpy>=1.14.3,<2.2.0',  # Update pyproject.toml as well.
```

Comment:
please update toml as requested in the comment.

Reply:
https://github.com/apache/beam/pull/32745